unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityList/EntityListAdapter.cs

```diff
@@ -63,12 +63,16 @@ void SetInfo(DCLBuilderInWorldEntity entityToEdit)
     {
         if (this != null)
         {
-            nameTxt.text = entityToEdit.rootEntity.entityId;
-
-            if(string.IsNullOrEmpty(entityToEdit.descriptiveName))
+            if (string.IsNullOrEmpty(entityToEdit.descriptiveName))
+            {
                 nameIF.text = entityToEdit.rootEntity.entityId;
+                nameTxt.text = entityToEdit.rootEntity.entityId;
+            }
             else
+            {
                 nameIF.text = entityToEdit.descriptiveName;
```

Comment:
What `IF` of `nameIF` stands for? If its trying to abbreviate something it'd be more clear if we could just use the full words.

Reply:
InputField. I will change it if it's not clear enough